src/people/hiveChat/ActionArtifactRenderer.tsx

```diff
@@ -48,36 +69,65 @@ export const ActionArtifactRenderer: React.FC<ActionArtifactRendererProps> = obs
     const content = actionArtifact.content as ActionContent;
 
     const hasButtonOptions =
-      content.options &&
-      content.options.some(
-        (option) => option.action_type === 'button' || option.action_type === 'chat'
-      );
+      content.options && content.options.some((option) => option.action_type === 'button');
```

Comment:
@MuhammadUmer44 this change is not clear to me. 
@MahtabBukhari would this affect the Chat Response rendering?